commands/bot.py

```diff
@@ -2,17 +2,19 @@
 
 from annotation.futebot import command
 from util.commands import Commands
-from util.helpers import format_params
+from util.helpers import embed_commands
 
 
 @command(desc="List all commands")
-async def listall(ctx):
+async def listall(ctx, grep=None):
 
     commands = Commands.get_instance().dictionary
-    embed = Embed(title="Commands list", color=0x00ff75)
 
-    for cmd in sorted(commands.keys()):
-        embed.add_field(name=".{} {}".format(cmd, format_params(commands[cmd]['params'])),
-                        value=commands[cmd]['description'], inline=False)
+    if grep is None:
```

Comment:
I'd suggest to check for the case we actually want to catch.
```suggestion
    if grep:
```
If `grep` contains something, we should do the filtering logic and send the embed object via `ctx.send` and then call a return still inside the `if` block.

And outside we would proceed normally, so no need for writing the else block, and just leave it for indentation.